docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+services:
+  app:
+    image: dhis2/core:${DHIS2_VERSION}
+    volumes:
+      - dhis2:/opt/dhis2/
+      #- ./config/dhis2/log4j2.xml:/opt/dhis2/log4j2.xml:ro
```

Comment:
Can we uncomment this line?

Reply:
I did see a change in the output when including the file. Which is why it's commented. I'm wondering if we can remove both the file and that line?